qt/aqt/webview.py

```diff
@@ -412,7 +421,8 @@ def stdHtml(
         # print(html)
         self.setHtml(html)
 
-    def webBundlePath(self, path: str) -> str:
+    @classmethod
```

Comment:
This looks like an unrelated change?

Reply:
I am using this function upstairs here:
```
File: anki/qt/aqt/webview.py
70:     def javaScriptConsoleMessage(self, level, msg, line, srcID):
71:         # not translated because console usually not visible,
72:         # and may only accept ascii text
73:         srcID = srcID.replace(AnkiWebView.webBundlePath("/"), "")
74:         if level == QWebEnginePage.InfoMessageLevel:
```

Reply:
Now I am not using it anymore because I set to use a regex as it should be a more reliable solution in case of future updates. Then, I kept the function as a classmethod because it can be a classmethod.

Reply:
Sorry, I missed that you were using it above.